src/jsrs_parser.cc

```diff
@@ -150,6 +151,11 @@ static bool GetType(const char* begin, const char* end, Type* type) {
       }
       break;
     }
+    case 'N':
+    case 'I': {
+      *type = Type::kNumber;
```

Comment:
@aqrln, why'd you put an ellipsis in the end of your comment?
Also, the problem stated by @lundibundi has nothing to do with this line, it is just incorrect usage of the `strtol` function, and I will fix this.